<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>508</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>686</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Registrar estudiante</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>661</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="frm_container">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>60</number>
     </property>
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_student_code">
       <property name="text">
        <string>Código de estudiante</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txt_student_code">
       <property name="inputMethodHints">
        <set>Qt::ImhPreferNumbers</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_fullname">
       <property name="text">
        <string>Nombre completo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txt_fullname"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_birthdate">
       <property name="text">
        <string>Cumpleaños</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="de_birthdate">
       <property name="styleSheet">
        <string notr="true">height: 35px;
padding: 0px 5px;
border: 1px solid #ccc;
border-radius: 5px;</string>
       </property>
       <property name="locale">
        <locale language="Spanish" country="Mexico"/>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDate|Qt::ImhPreferNumbers</set>
       </property>
       <property name="displayFormat">
        <string>d/M/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2000</year>
         <month>2</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_password">
       <property name="text">
        <string>Contraseña</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txt_password">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btn_register">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #3a0ca3;
font-weight:bold;
border: none;
height: 35px;
color:white;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Registrarme</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Registro de estudiante</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
